src/Pages/Music/Videos/Videoid_Int.elm

```diff
@@ -0,0 +1,111 @@
+module Pages.Music.Videos.Videoid_Int exposing (Model, Msg, Params, page)
+
+import Element exposing (..)
+import Shared exposing (sendActions)
+import Spa.Document exposing (Document)
+import Spa.Page as Page exposing (Page)
+import Spa.Url exposing (Url)
+import WSDecoder exposing (AlbumObj, ArtistObj, SongObj, VideosObj)
+
+
+page : Page Params Model Msg
+page =
+    Page.application
+        { init = init
+        , update = update
+        , subscriptions = subscriptions
+        , view = view
+        , save = save
+        , load = load
+        }
+
+
+
+-- INIT
+
+
+type alias Params =
+    { videoid : Int }
+
+
+type alias Model =
+    { artist_list : List ArtistObj
+    , videoid : Int
+    , videos : Maybe VideosObj
+    , album_list : List AlbumObj
+    , videos_list : List VideosObj
+    }
+
+
+init : Shared.Model -> Url Params -> ( Model, Cmd Msg )
+init shared { params } =
+    ( { videoid = params.videoid
+      , videos = getVideos params.videoid shared.videos_list
+      , album_list = shared.album_list
+      , artist_list = shared.artist_list
+      , videos_list = shared.videos_list
+      }
+    , Cmd.none
+    )
+
+
+checkVideoId : Int -> VideosObj -> Bool
+checkVideoId firstId video =
+    if firstId == video.videoid then
+        True
+
+    else
+        False
+
+
+getVideos : Int -> List VideosObj -> Maybe VideosObj
+getVideos id videolist =
+    List.head (List.filter (\videos -> checkVideoId id videos) videolist)
+
+
+
+-- UPDATE
+
+
+type Msg
+    = SetCurrentlyPlaying SongObj
+
+
+update : Msg -> Model -> ( Model, Cmd Msg )
+update msg model =
+    case msg of
+        SetCurrentlyPlaying song ->
```

Comment:
While `song` is not wrong, it might be nicer to name this `video`

Reply:
done.